src/lib/components/layout/QLayout.svelte

```diff
@@ -1,4 +1,11 @@
 <script context="module" lang="ts">
+  import { createStyles, createClasses } from "$lib/utils";
+  import { setContext } from "svelte";
+  import type { Readable } from "svelte/motion";
```

Comment:
1. You probably want `svelte/store`, no?
2. Can we group `import type` statements above each other? I prefer to put them last, example:
```ts
import ...
import type ...
import type ...
```